cpp/Ice/config/Client.cpp

```diff
@@ -12,11 +12,14 @@ int
 main(int argc, char* argv[])
 {
     // Figure out my name.
+#ifdef _MSC_VER
+    size_t requestedSize;
+    char buffer[256];
+    getenv_s(&requestedSize, buffer, sizeof(buffer), "USERNAME");
```

Comment:
in theory we should check the return value for this function. It can return an error...